lib/libzpool/uzfs_mgmt.c

```diff
@@ -533,7 +537,7 @@ uzfs_zvol_create_cb(const char *ds_name, void *arg)
 	error = uzfs_dataset_zv_create(ds_name, &zv);
 	if (error) {
 		/* happens normally for all non-zvol-type datasets */
-		return (error);
+		return (0);
```

Comment:
can we change the return type of this function as `void` since in every case we are returning `0`.

Reply:
we can't change signature as it is called from dmu_objset_find_impl which have fixed prototype

Reply:
ok